websites/S/Sparked Host/dist/metadata.json

```diff
@@ -0,0 +1,33 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.0",
+    "author": {
+      "name": "harrydev",
```

Comment:
i'm 100% sure that it isn't your discord name 😉

Reply:
Fixed